### ISSUE TITLE
NCAR derecho

### ISSUE THREAD

**Issue by dabail10** [CICE-Consortium/CICE#844]
cheyenne is going away in December. @apcraig has been doing some initial testing on derecho, our new machine. The directory structure is also going to change as of Feb 2024. Key points:

1. /glade/p and /glade/collections are going away.
2. /glade/campaign is mounted on cheyenne and derecho as a normal filesystem
3. The current CICE and Icepack inputdata is on /glade/p/cesm/pcwg_dev. This will need to move to campaign. I propose we use:

/glade/campaign/cesm/development/pcwg

4. The standard compilers on derecho will be different. intel, nvhpc, gnu, and cray.

**Comment by apcraig** [comment]
I wonder if NCAR can create some links for /glade/p/cesm pointing to campaign space.  We can certainly update Cheyenne and Derecho ports, but it would be helpful to users of older versions of the code to have the links in place as well.

**Comment by apcraig** [comment]
I have added Derecho ports for intel, intelclassic, inteloneapi (only working for Icepack), gnu, cray, nvhpc, see #863.  Inputdata is still pointing at /glade/p.

@dabail10 reported separately,

"By the way, it does sound like we can maintain a link from /glade/p/cesm to campaign. However, any new derecho settings should point to campaign. I have set up the directories there as:

/glade/campaign/cesm/development/pcwg/CICE_data and Icepack_data"

We should update the Derecho inputdata paths at some point.
